lifecycle-operator/main.go

```diff
@@ -347,6 +359,10 @@ func main() {
 		setupLog.Error(err, "unable to set up ready check")
 		os.Exit(1)
 	}
+
+	// Set the metric value as soon as the operator starts
+	keptnLifecycleActive.Add(context.Background(), 1)
+	setupLog.Info("Keptn lifecycle-operator is alive")
```

Comment:
I think @thisthat mentioned to have this log before line 364 🤔

Reply:
Oh my bad I thought it was after line 364. I will make the fix now